src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -541,6 +648,12 @@ private static IEnumerable<double> LazySequence(double records, double start, do
         public static FormulaValue Abs(IRContext irContext, NumberValue[] args)
         {
             var arg0 = args[0];
+            
+            if (arg0 == null) 
+            {
+                return new BlankValue(irContext);
+            }
+
```

Comment:
`Abs(Blank)` should return 0 (blank values as input to functions in numeric context should be coerced to 0)

Reply:
Fixed